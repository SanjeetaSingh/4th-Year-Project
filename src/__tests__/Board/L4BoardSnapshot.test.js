import renderer from 'react-test-renderer'

import Level4 from '../../Level4/L4-Board/L4-Board'

it('snapshot test level 14', () => {
    const tree =  renderer.create(<Level4 />).toJSON()
    expect(tree).toMatchSnapshot();
});
