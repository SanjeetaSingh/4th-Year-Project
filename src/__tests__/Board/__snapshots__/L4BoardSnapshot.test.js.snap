// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test level 14 1`] = `
<div
  className="container"
>
  <div
    className="boardcontainer4"
  >
    <div
      className="board4"
    >
      <div
        className="box"
        data-col={0}
        data-row={0}
      >
        <img
          alt="f"
          class="food"
          id="foodOne"
          src="assets/food.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={0}
      >
        <img
          alt="c"
          class="cat"
          id="cat"
          src="assets/cat.png"
        />
      </div>
      <div
        className="box"
        data-col={2}
        data-row={0}
      />
      <div
        className="box"
        data-col={3}
        data-row={0}
      >
        <img
          alt="h"
          class="hole"
          id="holeTwo"
          src="assets/hole.png"
        />
      </div>
      <div
        className="box"
        data-col={4}
        data-row={0}
      >
        <img
          alt="d"
          class="dog"
          id="dog"
          src="assets/dog.png"
        />
      </div>
      <div
        className="box"
        data-col={0}
        data-row={1}
      >
        <img
          alt="h"
          class="hole"
          id="holeOne"
          src="assets/hole.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={1}
      />
      <div
        className="box"
        data-col={2}
        data-row={1}
      />
      <div
        className="box"
        data-col={3}
        data-row={1}
      />
      <div
        className="box"
        data-col={4}
        data-row={1}
      />
      <div
        className="box"
        data-col={0}
        data-row={2}
      />
      <div
        className="box"
        data-col={1}
        data-row={2}
      />
      <div
        className="box"
        data-col={2}
        data-row={2}
      />
      <div
        className="box"
        data-col={3}
        data-row={2}
      />
      <div
        className="box"
        data-col={4}
        data-row={2}
      />
    </div>
  </div>
  <div
    className="bottom-panel"
  >
    <div
      class="border"
    >
      <div
        class="level2Contain"
      >
        <h2>
          Level 4:
        </h2>
        <div
          class="speech"
        >
          Your aim for this level is to help the dog get past the obstacles present. You HAVE to use the if statement that will allow you to pass the cat. In this level you can jump over the hole to avoid restarting and you must use the else statement. You have to use the same number or less number of commands mentioned in the top right corner.
          <br />
           Good luck! 
          <br />
           (To learn more about if statements click the i below)
        </div>
        <div
          class="borderPanel"
        >
          <div
            class="toppart"
          >
            <div
              class="containing"
            >
              <p
                class="number"
                data-testid="counter"
                id="count"
              >
                0/8
              </p>
            </div>
            <p
              class="titles"
            >
              Enter Sequence:
            </p>
            <p
              class="lists"
              id="action"
            />
          </div>
        </div>
        <div
          class="outside"
        >
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Left
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Up
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Down
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Right
          </button>
          <div
            class="buttons-wrapper"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
            >
              If Statement
            </button>
          </div>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Bark
          </button>
          <div
            class="buttons-wrapper"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              Else If
            </button>
          </div>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Jump
          </button>
          <button
            class="buttonSub"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
             Submit
          </button>
          <button
            class="buttonClear"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Clear
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
